### ISSUE TITLE
New migration file

### ISSUE THREAD

**Issue by vnorbix** [jrief/djangocms-cascade#116]
When I run makemigrations in my project, I got a new migration file for cascade:
```
Migrations for 'cmsplugin_cascade':
  0004_auto_20151118_1231.py:
    - Delete model HorizontalRulePluginModel
    - Delete model PanelGroupPluginModel
    - Delete model PanelPluginModel
    - Delete model SimpleWrapperPluginModel
    - Create proxy model BootstrapAccordionPanelPluginModel
    - Create proxy model BootstrapAccordionPluginModel
    - Create proxy model BootstrapPanelPluginModel
```
I'm using django 1.7, cascade 0.6.2. Is this migration file left off the current version, or something else is going on?

**Comment by jrief** [comment]
You can ignore that file. It's created by Django migrations and only renames the models into proxy models.
Please recheck if what I write here is correct. I have to find out how to prevent the generation of this file.

**Comment by frnhr** [comment]
The file will cause problems when a "real" 0004 migration comes out, with a future update to djangocms-cascade. At that time, that new migration won't be run, because Django will already have recorded the app migrated up to 0004. Just saying :)

**Comment by jrief** [comment]
OK, then I will add that.
I'm not too happy about the way Django handles migrations. Cascade auto-generates a lot of proxy models, which cause those migration files. If there is a recipe to inhibit this, I'd be happy.

**Comment by rfleschenberg** [comment]
It might be worthwhile to look at Django's `squashmigrations` feature (https://docs.djangoproject.com/en/1.9/topics/migrations/#squashing-migrations) to reduce the number of migration files. However, IIUC, this must be coordinated with release schedules and user documentation.

In general, we obviously must handle migrations with care, to avoid breaking existing installations. I think that if in doubt, it is better to go with what `makemigrations` does. Better to have a few more migration files than to break user installations :)

**Comment by rfleschenberg** [comment]
Fixed in PR #142.

There does not seem to be a clean way to suppress these kinds of migrations, so we have to accept them, at least for now. Maybe we can still figure out some way. Maybe we should ask for opinions on django-users.
